src/components/material/MaterialHeader.tsx

```diff
@@ -153,9 +154,11 @@ const MaterialHeader: React.FC<MaterialHeaderProps> = ({
                     materialTitleId={materialTitleId}
                   />
                 </div>
-                <MaterialAvailabilityText
-                  manifestations={selectedManifestations}
-                />
+                {!userIsAnonymous() && (
```

Comment:
This will have to be changed to the new "isUserAnonymous()" right?

Reply:
Yes when https://github.com/danskernesdigitalebibliotek/dpl-react/pull/423 is merged